src/game_objects/game_world_entity.py

```diff
@@ -57,59 +57,57 @@ def update_movement_velocity(self, game_state, game_world_state):
     def update_hitbox(self):
         self.state[const.HITBOX] = self.state[const.HITBOX_CONFIG][self.state[const.STATE]]
 
-    def apply_collision_detection(self, game_state):
-        new_hitbox_data = self.state[const.HITBOX_CONFIG].get(self.state[const.STATE])
-        hitbox_pos = new_hitbox_data[0]
-        hitbox_size = new_hitbox_data[1]
 
+    def apply_collision_detection(self, game_state):
+        hitbox_pos, hitbox_size = self.state[const.HITBOX_CONFIG].get(self.state[const.STATE])
```

Comment:
I'll be sure to leave this style of variable declaration in from now on.

Reply:
its great for unpacking tuples and lists, i think i did the same thing when building the platform hitboxes in the hit detection logic